src/GAP/gap_to_oscar.jl

```diff
@@ -0,0 +1,41 @@
+## conversions of GAP objects to Oscar objects
+## (extends the conversions from GAP.jl's `src/gap_to_julia.jl` and
+## `src/constructors.jl`, where low level Julia objects are treated)
+
+import GAP: gap_to_julia
+import Nemo: FlintIntegerRing, FlintRationalField, MatrixSpace, fmpz, fmpq, fmpz_mat, fmpq_mat
+
+## GAP integer to `fmpz`
+GAP.gap_to_julia(::Type{fmpz}, obj::Union{GAP.GapObj, Int64}) = fmpz(BigInt(obj))
```

Comment:
```suggestion
GAP.gap_to_julia(::Type{fmpz}, obj:: GAP.GapObj) = fmpz(BigInt(obj))
GAP.gap_to_julia(::Type{fmpz}, obj:: Int64) = fmpz(obj)
```
This avoids converting machine integers to bigint then to fmpz